evidence/serban300/0001-2024-H1-rank-1-retention.md

```diff
@@ -0,0 +1,46 @@
+# Evidence-0000: Retention at to Rank 1
+
+|                 |              |
+| --------------- |--------------|
+| **Report Date** | 2024/07/03   |
+| **Submitted by**| Serban Iorga |
+
+
+## Member details
+
+- Matrix username: `@serban.iorga:parity.io`
+- Polkadot address: `14oHMAJ5btnDCusHrTWraw1wTsLJwZeqPDLxusm1R1Zh3Vxa`
+- Current rank: `1`
+- Date of initial induction: `2024/04/22`
```

Comment:
serban300's [initial induction](ipfs://bafybeidcdu2mcvg3dwb5atfik3oz7svsimznjfqz4tmu4gmiprauqvfxei/#/extrinsics/3373411-2) was on 12th March 2024.

Reply:
Thanks ! Fixed. I thought I had to use the date when the call was executed.